py/test_runner.py

```diff
@@ -110,16 +39,14 @@ def run_test(args):  # pylint: disable=too-many-branches,too-many-statements
     util.configure_kubectl(project, zone, cluster_name)
   util.load_kube_config()
 
-  api_client = k8s_client.ApiClient()
-  masterHost = api_client.configuration.host
-
   t = test_util.TestCase()
   t.class_name = "tfjob_test"
-  namespace, name, env = ks_util.setup_ks_app(args)
-  t.name = os.path.basename(name)
+  t.name = os.path.basename(args.test_method)
 
   start = time.time()
 
+  module = import_module("py." + args.test_module)
```

Comment:
Instead of reusing test_runner.py for every test, should we create a suitable base class for our unittests and main function and then add that to the appropriate module and then invoke each module directly?

So the pattern would be something like

class ShutdownPolicyTests(test_util.TestCase):
   def test_shutdown_chief(self):
       ...
   def test_shutdown_no_chief(self):
       ...
if __name__ == "__main__":
   test_runner.main()

I think the main advantage is that our tests start to look more like regular Py unittests.

Reply:
Done.